doc/getting_started/index.rst

```diff
@@ -3,319 +3,372 @@
 Getting Started Guide
 #####################
 
-Follow this guide to set up a :ref:`Zephyr <introducing_zephyr>` development
-environment, then build and run a sample application.
+Follow this guide to get a quick start with Zephyr development where
+you'll:
 
-.. tip::
-
-   Need help with something? See :ref:`help`.
+#. **Set up a command-line development environment** for Linux*, macOS*, or
+   Windows*, with required compiler and build-system tools,
+#. **Get the sources**,
+#. **Build, flash, and run** a sample application on your target board.
 
 .. _host_setup:
 
-Install Host Dependencies
-*************************
+Set up development environment
+******************************
+
+Select and Update OS
+====================
+
+Zephyr development depends on up-to-date releases of common build system
+tools so first, make sure you're development system OS is updated:
+
+.. tabs::
 
-.. _python-pip:
+   .. group-tab:: Linux
```

Comment:
This instructions being only for Ubuntu, I wonder if it would be good to make it a bit more discoverable for other distros. Maybe a link to docs/getting_started/installation_linux.html, maybe more tabs(?)

Reply:
There is no such thing as "Linux". Linux runs my smartphone and my wifi router and I doubt they will ever be able to build Zephyr. Please rename the "Linux" tab to "Ubuntu 12.34" and mention that other Debian-based distros should work the same for the most part - but even there your mileage may vary. 

I've been building Zephyr on Fedora with practically no issue *except* for the main one: manually "translating" the Getting Started Guide. It's fine, I'm used to it.

Reply:
I agree, it just doesn't work to write "Linux" here and then have instructions for Ubuntu. It will be confusing for users. Either write "Ubuntu" and then also add a link to the info for the rest of Linux derivatives or change this to contain the instructions for all distros.